lua/gatekeeper/dylib.lua

```diff
@@ -81,6 +81,28 @@ int gk_flush_flow_table(const char *src_prefix,
 
 c = ffi.C
 
+local function fib_action_to_str(fib_action)
```

Comment:
Drop the qualifier `local`, so someone following the example of `print_fib_dump_entry()` and/or `print_neighbor_dump_entry()` can call `fib_action_to_str()` as well.